styles/globals.css

```diff
@@ -1,3 +1,14 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer base {
+    html {
+      font-family: Inter, system-ui, sans-serif;
+      /*font-family: Montserrat, system-ui, sans-serif;*/
```

Comment:
try to remove code comments, either leave it if needed or take out

Reply:
done